docs/Dev.md

```diff
@@ -1,7 +1,37 @@
 # Developer guide
 
+- [Setitng up Go workspaces](#setting-up-go-workspaces)
 - [How to release](#how-to-release)
 - [Updating OT core](#updating-ot-core)
+- [Running Tracing E2E tests](#running-tracing-e2e-tests)
+
+## Setting up Go workspaces
+
+This repository contains multiple Go packages with their own dependencies. Some IDEs
+(VS Code for example) do not like this kind of setup and demand that you work on each 
+package in a separate workspace. As of Go 1.18 this can be solved by configuring
+a single Go workspace covering all the packages. This can be done by adding a `go.work`
+file to the repository root:
```

Comment:
Perhaps add a link to the relevant release 1.18 section: https://tip.golang.org/doc/go1.18#go-work ?

Reply:
Done